lib/Software/Security/Policy/Individual.pm

```diff
@@ -81,6 +97,9 @@ a git url where the most current security policy can be found.
 
 the URL where you can report security issues.
 
+The B<report_url>, if defined, will override both the B<maintainer>
+and B<security_contact> (if defined)
+
```

Comment:
Likewise as above,

> If the `report_url` field is defined, it will be presented as the _primary_ contact point for security issues. The `maintainer` field will continue to be presented as a contact point for other issues.
> If unset, the `maintainer` remains presented as the primary contact point for all issues.
> If the `security_contact` field is also set, it will be offered as an _secondary_ contact point for security issues.

Does this make sense?